sdv/data_processing/data_processor.py

```diff
@@ -292,6 +336,25 @@ def fit(self, data):
         self._fit_hyper_transformer(constrained, columns_created_by_constraints)
         self.fitted = True
 
+    def generate_primary_keys(self, num_rows):
+        """Generate the columns that are identified as ``primary keys``.
+
+        Args:
+            num_rows (int):
+                Number of rows to be created. Must be an integer greater than 0.
+
+        Returns:
+            pandas.DataFrame:
+                A data frame with the newly generated primary keys of the size ``num_rows``.
+        """
+        if self._hyper_transformer.field_transformers.get(self._primary_key) is None:
+            return pd.DataFrame({self._primary_key: np.arange(num_rows)})
+
+        return self._hyper_transformer.create_anonymized_columns(
+            num_rows=num_rows,
+            column_names=[self._primary_key]
```

Comment:
I believe `self._primary_key` can be a tuple, since it can be set in line 248. In such cases we would want to pass each of them separately no?